guava-testlib/src/com/google/common/collect/testing/IteratorTester.java

```diff
@@ -50,6 +50,10 @@
  * verify() method, which is called <em>after</em> each sequence and is guaranteed to be called
  * using the latest values obtained from {@link IteratorTester#newTargetIterator()}.
  *
+ * <p>The value you pass to the parameter {@code steps} should be greater than the length of your
+ * iterator, so that this class can check that your iterator exhibits the same edge-case behavior as
```

Comment:
What do you think of just saying "...that your iterator exhibits the correct behavior when it is exhaused?"

Reply:
Yep, that sounds good to me. :+1:

Reply:
I'm not sure if you were expecting me to update the docs or if you were happy to do it yourself?

Reply:
Sorry, I'm taking care of it. The PR is out for internal review.

Reply:
Cheers @cpovirk! 👍